tests/test_coordinates.py

```diff
@@ -194,6 +194,24 @@ def test_len(self):
         """len of Coords should return number of 3-vectors"""
         self.assertEqual(len(self.cvals), 2)
 
+    def test_SPH_output_noconvert(self):
+        """SPH is a synonym of GEO in spherical"""
+        self.cvals.ticks = Ticktock(['2002-02-02T12:00:00', '2002-02-02T12:00:00'], 'ISO')
```

Comment:
Is this actually being used? I can see if self.cvals isn't, it's still worth keeping in the same class for clarity, but I'm not sure why apply ticks to it.

Reply:
It's a vestigal setup where I started writing the test by copy-pasting another test. I'll condense it.